### PR TITLE
feat: [CCM-7009]: RDS enhancements

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/75-ce/components/COGatewayList/components/ComputeType.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <Layout.Horizontal spacing="small" className={className}>
+      {isK8sRule ? (
+        <Icon name="app-kubernetes" size={21} />
+      ) : isEcsRule ? (
+        <Icon name="service-ecs" size={21} />
+      ) : isRdsRule ? (
+        <Icon name="aws-rds" size={21} />
+      ) : (
+        <img src={getIcon()} alt="" aria-hidden />
+      )}
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayList/components/ComputeType.tsx:56]: knit: Better to use a function with simple if blocks. Improves readability

**Review by jenil-harness — APPROVED** [review]

**Review by jyotiarora-harness — DISMISSED** [review]

**Review by rajarshee-harness — APPROVED** [review]

**Review by navaneeth — APPROVED** [review]

**Review by rajarshee-harness — DISMISSED** [review]

**Comment by rishik30** [comment]
retrigger titlecheck

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger prettier

**Comment by rishik30** [comment]
etrigger typecheck

**Review by rajarshee-harness — APPROVED** [review]

**Comment by rishik30** [comment]
trigger cypress

**Review by navaneeth — APPROVED** [review]

**Pull request merged**